src/stravalib/model.py

```diff
@@ -895,7 +896,53 @@ class SummaryActivity(MetaActivity, strava_model.SummaryActivity):
     )(check_valid_location)
 
 
-class Activity(
+class Trend(BaseModel):
+    speeds: Optional[list[float]] = None
```

Comment:
undocumented

Reply:
removed this as well! that point is valid. no need to add extra undocumented attrs if users don't need them.